com.unity.render-pipelines.universal/Runtime/Passes/ScriptableRenderPass.cs

```diff
@@ -150,6 +150,16 @@ public RenderTargetIdentifier depthAttachment
             get => m_DepthAttachment;
         }
 
+        public RenderBufferStoreAction colorStoreAction
+        {
+            get => m_ColorStoreActions[0];
+        }
```

Comment:
It seems limited API that you can't get the MRT store action.

Reply:
agree, will fix

Reply:
fixed in the latest commits